src/ansys/dpf/gate/generated/field_definition_capi.py

```diff
@@ -184,6 +184,35 @@ def csfield_definition_get_shell_layers(res):
 		if errorSize.value != 0:
 			raise errors.DPFServerException(sError.value)
 		return res
+	
+	@staticmethod
```

Comment:
this is supposed to be generated, how did you do it?

Reply:
@cbellot000  this is fixed, I used the generated code and removed the added one